tut/stream-client.md

```diff
@@ -74,7 +79,8 @@ Create a `demo.html` file inside `/srv/stream-client`:
     });
     stream.connect();
     </script>
-    <!--To display the video element correctly, always specify both the height and the width attributes-->
+    <!--To display the video element correctly,
+    always specify both the height and the width attributes-->
```

Comment:
```suggestion
    <!--To display the video element correctly,
        always specify both the height and the width attributes-->
```
To make the text aligned. It's not necessary, I just think it looks a bit better, but I'm fine either way. What do you think?